### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by S-BLH**
1. created a repo
2. cloned it to my local machine
3. configured the linters
4. added the html,css and js files
5. added the codes to create a telephone number validator

### CODE REVIEW

**Comment by latifa-wakili** [comment]
@S-BLH good job!
#- your codes are clean 
#- great style
#- no linters error
#-great responsive 
now mention teacher for thier review 

good Job! keep it up sumita jan

**Comment by S-BLH** [comment]
@ahzamir ostad please review my project

File: script.js
Comment (ahzamir) [inline script.js:1]: - [OPTIONAL] - Try to add your Event Listeners in while DOM content is loading, it helps to add all of your event listeners to be ready to the action right when your page loads and prevents any side effects below is an example to help you do that:

```javascript
document.addEventListener('DOMContentLoaded', () => {
  //In here add all of your addEventListener functions
  // don't forget to add the constants that you need in your event listeners here as the scope of this one is higher than other codes
  
  // Here add all of you code that belongs to event listeners
})
```

**Review by ahzamir — APPROVED** [review]
### Hi @S-BLH 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

Check the [OPTIONAL] changes below implement them and then merge your branch.

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Telephone Number Validator project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**